tests/test_D1_model_components.py

```diff
@@ -191,39 +218,50 @@ def test_transformer_fix_cap_single_busses(self):
         )
 
     def test_transformer_fix_cap_multiple_input_busses(self,):
-        ## todo fix after decision on busses see todo in func above
-        # dict_asset = self.dict_values[ENERGY_CONVERSION ][
-        #     "transformer_fix_multiple_input_busses"
-        # ]
-        #
-        # D1.transformer(
-        #     model=self.model,
-        #     dict_asset=dict_asset,
-        #     transformer=self.transformers,
-        #     bus=self.busses,
-        # )
-        #
-        # # one output bus and two input busses
-        # assert len([str(i) for i in self.model.entities[-1].outputs]) == 1
-        # assert len([str(i) for i in self.model.entities[-1].inputs]) == 2
-        #
-        # # checks done with helper function (see func for more information)
-        # self.helper_test_transformer_in_model_and_dict(optimize=False, dict_asset=dict_asset)
-        pass
+        dict_asset = self.dict_values[ENERGY_CONVERSION][
+            "transformer_fix_multiple_input_busses"
+        ]
+
+        D1.transformer(
+            model=self.model,
+            dict_asset=dict_asset,
+            transformer=self.transformers,
+            bus=self.busses,
+        )
+
+        # one output bus and two input busses
+        assert len([str(i) for i in self.model.entities[-1].outputs]) == 1
+        assert len([str(i) for i in self.model.entities[-1].inputs]) == 2
+
+        # checks done with helper function (see func for more information)
+        self.helper_test_transformer_in_model_and_dict(
+            optimize=False, dict_asset=dict_asset
+        )
 
     def test_transformer_fix_cap_multiple_output_busses(self):
-        pass
+        dict_asset = self.dict_values[ENERGY_CONVERSION][
+            "transformer_fix_multiple_output_busses"
+        ]
 
     ### tests for transformer with time dependent efficiency
     def test_transformer_efficiency_time_series_optimize_cap_multiple_input_busses(
         self,
     ):
         pass
```

Comment:
Is the "pass" before you execute the function `D1.transformer` intentional?

Reply:
code not up-to-date here